convert2rhel/unit_tests/subscription_test.py

```diff
@@ -146,6 +146,42 @@ def __call__(self, *args, **kwargs):
         return [namedtuple("Sub", ["pool_id", "sub_raw"])("samplepool", "Subscription description")]
 
 
+class PromptUserMocked(unit_tests.MockFunction):
+    def __call__(self, *args, **kwargs):
+        return True
+
+
+class TestCheckNeededReposAvailability(object):
+    def test_check_needed_repos_availability(self, monkeypatch, caplog):
+        monkeypatch.setattr(utils, "ask_to_continue", PromptUserMocked())
+        monkeypatch.setattr(subscription, "get_avail_repos", lambda: ["rhel_x", "rhel_y"])
+
+        avali_repos_message = "Needed RHEL repositories are available."
```

Comment:
Should "`avali`" throughout here be "`avail`"? (e.g.):
|Current|Suggested|
|-------|---------|
|`avali_repos_message`|`avail_repos_message`|
|`no_avali_repos_message`|`no_avail_repos_message`|

Reply:
Thank you for catching this for me, I will make the correct changes in the code!